core/src/main/java/com/exadel/aem/toolkit/core/authoring/models/Slider.java

```diff
@@ -55,6 +65,34 @@ public class Slider extends BaseModel {
     @ValueMapValue
     private boolean filled;
 
+    @ValueMapValue
+    private String startValue;
+
+    @ValueMapValue
+    private String endValue;
+
+    @ValueMapValue
+    private boolean ranged;
+
+    @ValueMapValue
+    private SliderItem[] items;
+
+    @ValueMapValue
+    private Map<String, String> itemsMap;
```

Comment:
Same applies here ) You cannot inject a map this way. Moreover, you don't event need to inject anything here because even if you managed to, you ignore the injected value and overwrite in in line 85

Reply:
Please consider naming this `items`. Because you'll probably delete the field at line 78 and you will have no other items

Reply:
Aah yes, youre right got it.

Reply:
Thanks!